sdcm/cluster.py

```diff
@@ -1406,6 +1406,11 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
                                          scylla_yaml_contents)
 
         if server_encrypt or client_encrypt:
+            try:
+                localrunner.run('openssl x509 -req -in data_dir/ssl_conf/example/db.csr -CA data_dir/ssl_conf/cadb.pem -CAkey data_dir/ssl_conf/example/cadb.key -CAcreateserial -out data_dir/ssl_conf/db.crt -days 365')
+                localrunner.run('openssl x509 -enddate -noout -in data_dir/ssl_conf/db.crt')
+            except Exception as ex:
+                self.log.error('Failed to update encryption key: %s', ex)
```

Comment:
we need to fail the test when the key can't be created

Reply:
and make this before we provision the instances, make a method calledd `update_certificates` and run it in setUp phase

Reply:
fixed.